CONTRIBUTING.md

```diff
@@ -96,6 +96,26 @@ you'll learn high-level context about the project's different components without
   the code as well as learning the submission and review process (which is a significant learning step if you've
   never submitted a PR before).
 
+## Uplifting the Theia dependencies version
+
+Should one be willing to contribute such an uplift, these are the steps to consider, while
+more may be necessary depending on the case. Here is an example used to uplift from version
+`1.34.1` to `1.34.2`:
+
+1. `cd theia` or go to the Theia repository using a local terminal.
```

Comment:
Can you make it 2, 3, 4, and so on since it is an ordered list?

Reply:
This shouldn't be necessary in Markdown.
- Using `1.` across leads to [automated numbering (increments) at rendering time](https://github.com/eclipse-cdt-cloud/theia-trace-extension/blob/1d0684e4113ada3ac3cfaaf7894c07a8938d65f0/CONTRIBUTING.md).
- It also allows for not having to manually maintain the said numbering (increments).